### ISSUE TITLE
Document packaging

### ISSUE THREAD

**Issue by ehuelsmann** [RexOps/rexify-website#104]
To be documented
* Project doesn't publish projects of its own
* Project publishes release-tarballs to create packages from
* Project helps packagers

**Comment by ehuelsmann** [comment]
Very much relates to #52 and #53. Fixing this one will allow closure of those two.

**Comment by ferki** [comment]
Yes, and I would argue that the whole installation page needs a major overhaul, _before_ the packing strategy gets announced. Just by skimming through it:
 - I believe the `curl | perl --sudo` method should be discouraged
 - rexops maintained repos can be cleaned up, mentioning distros which are already packaging rex on their own
 - the recommended method is proposed to be "install from CPAN", with the alternative being installing from source
 - for Mac OS X, homebrew can also be used

And we should really publish the recommended/preferred installation method _before_ the options go away. I'll create a separate issue for that.

**Comment by ferki** [comment]
Also depends on #116.
